nix/k.nix

```diff
@@ -4,7 +4,7 @@
 
 let
   unwrapped = mavenix.buildMaven {
-    name = "k-${version}";
+    name = "k-5.3.0";
```

Comment:
@goodlyrottenapple where was this `${version}` identifier being read from in the first place? I see on line 1 that `version` is imported.

I'm skeptical about reading this from the file `package/version`, because that file is populated/touched in the build process I think, or maybe it only stores the major/minor versions but not the patch.

So maybe we should just go back to the previous way of doing it for now? Is there anything wrong with hardcoding the version number here and having `version/version.sh` update it?